### DISCUSSION TITLE
[Feature] Be able to edit profile photo on the web gui

### DISCUSSION THREAD

**Discussion by red-0130** [immich-app/immich#3025] [Feature Request]
### The feature

Not really much of an issue but I noticed that the web GUI does not have the feature to modify the profile pic. So far, users can only change it through the mobile app.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by ResponsiblyUnranked** [comment]
I agree this would be a neat feature 😄

**Comment by bo0tzz** [comment]
When viewing a photo in the web ui you can set it as your profile picture 
![image](https://github.com/immich-app/immich/assets/11602424/55b04ba1-e244-4483-b71c-1a4e3db986e4)

**Reply by ResponsiblyUnranked** [reply]
Thanks!

**Reply by coltenkrauter** [reply]
Are there plans to streamline this process a bit more? It is intuitive to the little edit button on the profile picture... The one that apparently only lets users change the color. I would expect the `Select avatar color` to be expanded to allow me to upload an avatar photo right there.

**Comment by freekvh** [comment]
It's pretty funny because on my iPad it wants me to find a photo as an avatar, from my iPad library... But you know where all my photos are? Not in the iPad library ;)

I managed to set it with the "set as profile picture" though :)

Love you guys, keep it up!
